pkgs/tools/typesetting/tex/texlive/default.nix

```diff
@@ -100,11 +100,15 @@ let
               map (up: "${up}/${urlName}.tar.xz") urlPrefixes
             );
 
-      # Upstream refuses to distribute stable tarballs,
-      # so we host snapshots on IPFS or on our own servers.
-      # Common packages should get served from the binary cache anyway.
-      # See discussions, e.g. https://github.com/NixOS/nixpkgs/issues/24683
+      # The official tarballs for the current release are constantly receiving
+      # updates, so we can't use those directly. More stable (but less
+      # official) hosted snapshots need to be used instead. Ideally, for the
```

Comment:
I wouldn't say necessarily "less official" ;-) We don't host them on tug.org because the server is not strong enough, and as Karl and me are the "release managers", it as as much official as it can be.

Reply:
@norbusan I removed any mention of "official" and did s/maintainers/release managers/.

Reply:
THere are no **official** titles in TeX Live, just Karl and me doing the stuff, and because we have the GPG keys for the release, we are the maintainers, release managers, and whatever comes to your mind. I think "maintainer" is the best expression anyway. Sorry for the confusion. But leave it as you might fits best!